fe/fe-core/src/main/java/com/starrocks/catalog/CatalogRecycleBin.java

```diff
@@ -760,7 +760,11 @@ public void addTabletToInvertedIndex() {
                 }
             } // end for indices
         }
+    }
 
+    public void removeInvalidateReference() {
```

Comment:
Where will this function be called

Reply:
Good point! I forgot...just added the caller few seconds ago.